libs/react-design-system/src/immersionFacile/components/task/Task.tsx

```diff
@@ -0,0 +1,56 @@
+import { fr } from "@codegouvfr/react-dsfr";
+import Button, { type ButtonProps } from "@codegouvfr/react-dsfr/Button";
+import type { ReactNode } from "react";
+import { useStyles } from "tss-react/dsfr";
+import { semanticTitleToClassName, type TitleLevel } from "../../utils";
+import { taskStyles } from "./Task.styles";
+
+export type TaskProps = {
+  className?: string;
+  title: ReactNode;
+  titleAs: TitleLevel;
+  description: ReactNode;
+  buttonProps?: ButtonProps;
+  footer?: ReactNode;
+};
+
+export const Task = ({
+  title,
+  titleAs,
+  description,
+  className,
+  buttonProps,
+  footer,
+}: TaskProps) => {
+  const { cx } = useStyles();
+  const Title = titleAs;
+
+  return (
+    <section
```

Comment:
```suggestion
    <article
```
Mais c'est du détail